dart/src/main/java/com/f2prateek/dart/internal/InjectExtraProcessor.java

```diff
@@ -86,6 +87,20 @@
       } catch (IOException e) {
         error(typeElement, "Unable to write injector for type %s: %s", typeElement, e.getMessage());
       }
+
+      // Now write the IntentBuilder
+      try {
+        IntentBuilder intentBuilder = new IntentBuilder(extraInjector.getClassPackage(),
+            typeElement.getSimpleName()+INTENT_BUILDER_SUFFIX, extraInjector.getTargetClass(),
+            extraInjector.getInjectionMap());
+        JavaFileObject jfo = filer.createSourceFile(intentBuilder.getFqcn(), typeElement);
+        Writer writer = jfo.openWriter();
+        writer.write(intentBuilder.brewJava());
+        writer.flush();
```

Comment:
Existing code but it's a good point. @f2prateek should this be a separate issue?